scss/_pagination.scss

```diff
@@ -27,20 +27,22 @@
     outline: $pagination-focus-outline;
     box-shadow: $pagination-focus-box-shadow;
   }
+  
+  &.active,
+  .page-item.active & {
```

Comment:
`.active > & {` feels more accurate to me, taking direct parenthood into account while decreasing the specificity.

Reply:
Yes I agree with you. But this specificity it's like current code that we removed:

```scss
.page-item {
  //...
  &.active .page-link {
    z-index: 3;
    color: $pagination-active-color;
    @include gradient-bg($pagination-active-bg);
    border-color: $pagination-active-border-color;
  }
  //...
}
```

Anyway, I have make change as you suggest.